File: Hackwithinfy/pilesofstone.py
def ispossible(a,b,c,x,y):
    if (a+b+c) != (x + y):
        return "NO"

    elif ( y < min(a, min(b,c))) or (x < min(a, min(b,c))):
        return "NO"

    return "YES"


def main():

    test = int(input())

    for _ in range (test):

        a,b,c,x,y = map(int,input().split())

        result = ispossible(a,b,c,x,y)

        print(result)

main()

File: Hackwithinfy/pilesofstone.c
#include<stdio.h>
#define min(a, b) (((a) < (b)) ? (a) : (b))

const char *ispossible(long long int a, long long int b, long long int c, long long int x, long long int y)
{

    if ((a+b+c) != (x+y))
    {
        return "NO";
    }

    else if ( y < min(a,min(b,c)) || x < min(a, min(b,c)))
    {
        return "NO";
    }

    else
    {
        return "YES";
    }
}

int main()
{
    int test;
    scanf("%d",&test);

    while (test--)
    {
        long long int a, b, c, x, y;

        scanf("%lld%lld%lld%lld%lld",&a,&b,&c,&x,&y);

        printf("%s\n",ispossible(a,b,c,x,y));
    }

    return 0;
}
